sql/edition.sql

```diff
@@ -2,36 +2,19 @@
 -- Please see the included NOTICE for copyright information and
 -- LICENSE-APACHE for a copy of the license.
 
-CREATE OR REPLACE FUNCTION _timescaledb_internal.enterprise_enabled() RETURNS BOOLEAN
-AS '@MODULE_PATHNAME@', 'ts_enterprise_enabled' LANGUAGE C;
-
-CREATE OR REPLACE FUNCTION _timescaledb_internal.current_license_key() RETURNS TEXT
-AS '@MODULE_PATHNAME@', 'ts_current_license_key' LANGUAGE C;
-
 CREATE OR REPLACE FUNCTION _timescaledb_internal.tsl_loaded() RETURNS BOOLEAN
```

Comment:
I think we should also remove this function and function below or at least move them to some C function in the tests.

Reply:
Moved this stuff into test files.